extensions/amp-list/0.1/amp-list.js

```diff
@@ -490,17 +490,19 @@ export class AmpList extends AMP.BaseElement {
       return this.ssrTemplateHelper_.fetchAndRenderTemplate(
           this.element, request, /* opt_templates */ null, attributes);
     }).then(response => {
+      userAssert(response && !!response['html'],
+          'Server side response must be defined');
```

Comment:
Minor: A more detailed error message may be useful. E.g. 

```js
userAssert(..., 'Expected response with format {html: <string>}. Received:', response);
```

Reply:
done